.github/workflows/build_publish_lambda_layer.yml

```diff
@@ -65,6 +65,54 @@ jobs:
           name: ruby-layer-${{ matrix.arch }}.zip
           path: lambda/build/ruby-layer-${{ matrix.arch }}.zip
 
+  # extract the built layer from artifacts, then scan it with reverselab
+  reverselab_scan_layer:
+    needs:
+      - build_layer
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86_64
+          - arm64
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: extract layer zip from artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: ruby-layer-${{ matrix.arch }}.zip
+          path: lambda
+
+      - name: extract current solarwinds_apm version
+        run: |
+          APM_VERSION=$(grep "gem 'solarwinds_apm'" lambda/otel/layer/Gemfile | awk -F"'" '{print $4}')
+          echo "SOLARWINDS_APM_VERSION=$APM_VERSION" >> $GITHUB_ENV
+
+      - name: Scan build artifact on the Portal ${{ matrix.arch }}
+        id: rl-scan
+
+        env:
+          RLPORTAL_ACCESS_TOKEN: ${{ secrets.REVERSE_LAB_TOKEN }}
+
+        uses: reversinglabs/gh-action-rl-scanner-cloud-only@v1
+        with:
+          artifact-to-scan: ./lambda/ruby-layer-${{ matrix.arch }}.zip
+          rl-verbose: true
+          rl-portal-server: solarwinds
+          rl-portal-org: SolarWinds
+          rl-portal-group: SaaS-Agents-SWO
+          rl-package-url: solarwinds-apm-ruby/apm-ruby-lambda-layer-${{ matrix.arch }}@${{ env.SOLARWINDS_APM_VERSION }}
+          rl-submit-only: true
```

Comment:
Since the layer publish isn't blocked on the scan result, let's remove `rl-submit-only` and have the workflow get the actual scan status.

Reply:
Removed.